### DISCUSSION TITLE
Apollo Save Tool v1.0.2

### DISCUSSION THREAD

**Discussion by bucanero** [bucanero/apollo-ps2#26] [Announcements]
### Added

* Support additional PS2 save formats when importing to PS2 VMCs
  - Import to VMC - New supported formats: `.MAX`, `.CBS`, `.XPS`, `.SPS`
* Add support to load an offline save database (from `cdfs:/`, `mass:/`)
  - Note: requires `mass` drive to be available to unzip save files
  - Try the `sample-save-pack.zip` as a quick Offline DB pack

### Fixed

* Solve issue when exporting PS1 saves from memory cards

<hr /><em>This discussion was created from the release <a href='https://github.com/bucanero/apollo-ps2/releases/tag/v1.0.2'>Apollo Save Tool v1.0.2</a>.</em>

**Comment by chaiki5topo** [comment]
Hiya can this work on ps3???
